app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -181,8 +181,9 @@ description: |
     for subman_config_file in /etc/yum/pluginconf.d/subscription-manager.conf /etc/dnf/plugins/subscription-manager.conf; do
       if [ -f $subman_config_file ]; then
         egrep -q "^disable_system_repos=" $subman_config_file
+        egrep -q "^disable_system_repos\s?=" $subman_config_file
         if [ "$?" -eq 0 ]; then
-          sed s/^disable_system_repos=.*/disable_system_repos=1/ $subman_config_file >"${subman_config_file}.new"
+          sed s/^disable_system_repos\s?=.*/disable_system_repos=1/ $subman_config_file >"${subman_config_file}.new"
```

Comment:
at least on my system (with `sed-4.8-8.fc35.x86_64`), I needed to pass `-E`/`--regexp-extended` to `sed`, as otherwise the `\s?` doesn't work and the replacement doesn't happen at all.